terracumber_config/tf_files/SUSEManager-4.1-PRV.tf

```diff
@@ -0,0 +1,185 @@
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins-env-fail.txt"
+}
+
+variable "MAIL_FROM" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = "string"
+}
+
+variable "SCC_PASSWORD" {
+  type = "string"
+}
+
+variable "GIT_USER" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://metropolis.prv.suse.net/system"
+}
+
+
+module "cucumber_testsuite" {
+  source = "./modules/cucumber_testsuite"
+
+  product_version = "4.1-nightly"
+
+  // Cucumber repository configuration for the controller
+  git_username = var.GIT_USER
+  git_password = var.GIT_PASSWORD
+  git_repo     = var.CUCUMBER_GITREPO
+  branch       = var.CUCUMBER_BRANCH
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  
+  images = ["centos7", "opensuse150", "sles15sp1", "sles15sp2", "ubuntu1804"]
+
+  use_avahi = false
+  name_prefix = "suma-41-"
+  domain = "prv.suse.net"
+  from_email = "root@suse.de"
+
+  portus_uri = "portus.mgr.suse.de:5000/cucutest"
+  portus_username = "cucutest"
+  portus_password = "cucusecret"
+
+  mirror = "minima-mirror.prv.suse.net"
+  use_mirror_images = true
+  server_http_proxy = "galaxy-proxy.mgr.suse.de:3128"
+
+  host_settings = {
+    ctl = {
+      provider_settings = {
+        mac = "52:54:00:00:00:26"
+      }
+    }
+    srv = {
+      provider_settings = {
+        mac = "52:54:00:00:00:31"
+      }
+    }
+    pxy = {
+      provider_settings = {
+        mac = "52:54:00:00:00:27"
+      }
+    }
+    cli-sles12sp4 = {
+      image = "sles15sp1"
+      name = "cli-sles15"
+      provider_settings = {
+        mac = "52:54:00:00:00:22"
+      }
+    }
+    min-sles12sp4 = {
+      image = "sles15sp1"
+      name = "min-sles15"
+      provider_settings = {
+        mac = "52:54:00:00:00:33"
+      }
+    }
+    min-build = {
+      image = "sles15sp1"
+      provider_settings = {
+        mac = "52:54:00:00:00:30"
+      }
+    }
+    minssh-sles12sp4 = {
+      image = "sles15sp1"
+      name = "minssh-sles15"
+      provider_settings = {
+        mac = "52:54:00:00:00:24"
+      }
+    }
+    min-centos7 = {
+      provider_settings = {
+        mac = "52:54:00:00:00:25"
+      }
+    }
+    min-ubuntu1804 = {
+      provider_settings = {
+        mac = "52:54:00:00:00:28"
+      }
+    }
+    min-pxeboot = {
+      present = true
+      image = "sles15sp1"
+    }
+    min-kvm = {
+      image = "sles15sp1"
+      provider_settings = {
+        mac = "52:54:00:00:00:29"
+      }
+    }
+  }
+  provider_settings = {
+    pool = "ssd"
+    network_name = null
+    bridge = "br0"
+    additional_network = "192.168.43.0/24"
```

Comment:
The idea was to have private network IP ranges identical to the version. Was `192.168.41.0/24` impossible here?

Reply:
Done.